canonical/ubuntu-pro-client
# docs: add mermaid plugin and replace image

## Why is this needed?

Fixes: #2559

## Test Steps

Make sure the diagram is rendered correctly and is an appropriate replacement.
Make sure it works for each of `tox -e docs`, `tox -e docs-dev` and on readthedocs

## Checklist

 - [n/a] I have updated or added any unit tests accordingly
 - [n/a] I have updated or added any integration tests accordingly

## Does this PR require extra reviews?
